src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -113,12 +115,14 @@ export class LogStreamRegistry {
     public async updateLog(
         uri: vscode.Uri,
         headOrTail: 'head' | 'tail' = 'tail',
+        configuration: SettingsConfiguration,
```

Comment:
seems like `this.configuration` could be used, the parameter is not ever different. And it would be surprising if the parameter _was_ different, since the `LogStreamRegistry` already stores `this.configuration` , no?

Reply:
Done